modules/installation/pages/proxy-setup.adoc

```diff
@@ -221,25 +219,6 @@ If you do not use the reactivation key, you will need to re-register all the cli
 
 
 
-.Procedure: Replacing a Traditional Proxy and Keeping the Clients Registered
```

Comment:
Do we now have this info elsewhere?  I can imagine that users in the field still have traditionally connected proxies running and want to update or replace them.  But that's just a feeling.

Reply:
I believe as of 4.2 Proxy is purely Salt, so this procedure is redundant. I will double check and request review again. Traditional proxy remains in 4.1 documentation.

Reply:
Please check, I am not sure about this one either.